packages/ui/src/utils/setUserAgent/constants.ts

```diff
@@ -1,24 +1,66 @@
 import {
+  AuthUserAgentInput,
+  GeoUserAgentInput,
+  InAppMessagingUserAgentInput,
+  StorageUserAgentInput,
   setCustomUserAgent,
   SetCustomUserAgentInput,
   AuthAction,
+  GeoAction,
+  InAppMessagingAction,
+  StorageAction,
   Category,
 } from '@aws-amplify/core/internals/utils';
 
 // replace type once Category input types are available
 export const ACCOUNT_SETTINGS_INPUT_BASE: Omit<
-  SetCustomUserAgentInput,
+  AuthUserAgentInput,
   'additionalDetails'
 > = {
   apis: [AuthAction.DeleteUser, AuthAction.UpdatePassword],
   category: Category.Auth,
 };
 
-// replace type once Category input types are available
-export const IN_APP_MESSAGING_INPUT_BASE: Omit<
+export const AUTHENTICATOR_INPUT_BASE: Omit<
   SetCustomUserAgentInput,
   'additionalDetails'
 > = {
-  apis: undefined, // TODO
+  apis: [
+    AuthAction.SignUp,
+    AuthAction.ConfirmSignUp,
+    AuthAction.ResendSignUpCode,
+    AuthAction.SignIn,
+    AuthAction.FetchMFAPreference,
+    AuthAction.UpdateMFAPreference,
+    AuthAction.SetUpTOTP,
+    AuthAction.VerifyTOTPSetup,
+    AuthAction.ConfirmSignIn,
+    AuthAction.FetchUserAttributes,
+    AuthAction.SignOut,
+    AuthAction.ResetPassword,
+    AuthAction.ConfirmResetPassword,
+    AuthAction.FederatedSignIn,
+  ],
+  category: Category.Auth,
+};
+
+export const GEO_INPUT_BASE: Omit<GeoUserAgentInput, 'additionalDetails'> = {
+  category: Category.Geo,
+  apis: [GeoAction.SearchByText],
+};
+
+export const IN_APP_MESSAGING_INPUT_BASE: Omit<
+  InAppMessagingUserAgentInput,
+  'additionalDetails'
+> = {
+  apis: [InAppMessagingAction.SyncMessages],
   category: Category.InAppMessaging,
 };
+
+export const STORAGE_INPUT_BASE: Omit<
+  StorageUserAgentInput,
+  'additionalDetails'
+> = {
+  apis: [StorageAction.GetUrl, StorageAction.UploadData],
```

Comment:
Couple things here:
- these constants map to a connected component, not a JS category
- we don't need to include getUrl since StorageImage is not using setUserAgent

Reply:
Ok I was thrown off by AccountSettings because it's broken up into ChangePassword and DeleteUser.. is AccountSettings still considered a component or should I split that up too?